src/python/pants/reporting/streaming_workunit_handler.py

```diff
@@ -37,7 +37,11 @@ def end(self) -> None:
         # we report any workunits that were added after the last time the thread polled.
         workunits = self.scheduler.poll_workunits()
         for callback in self.callbacks:
-            callback(workunits=workunits, finished=True)
+            callback(
+                workunits=workunits["completed"],
```

Comment:
I think `workunits` should be renamed to `completed_workunits`. I know this is a breaking change, but I don't think this API is public yet, right?

Reply:
`started` already differentiates enough, imo.